system/lib/pthread/library_pthread.c

```diff
@@ -937,7 +937,6 @@ int EMSCRIPTEN_KEEPALIVE proxy_main(int argc, char** argv) {
       // Proceed by running main() on the main browser thread as a fallback.
       return __call_main(_main_arguments.argc, _main_arguments.argv);
     }
-    EM_ASM(noExitRuntime = true);
```

Comment:
how is this change related to PROXY_TO_PTHREAD?

Reply:
This EM_ASM line is what used to keep the runtime alive in PROXY_TO_PTHREAD build mode.

Reply:
I see, thanks!